releasenotes/notes/named-entity-extractor-component-8fd647ee748892ca.yaml

```diff
@@ -0,0 +1,3 @@
+---
+features:
+  - New NamedEntityExtractor component.
```

Comment:
We're trying to make the release notes more explicative, reason why we moved from PR titles to yaml definitions using `reno`, so we can use multiple paragraphs to explain a feature. Since we already have a great explanation in the PR description, I would copy it here in the release note, something like:
> Added a new extractor component, namely NamedEntityExtractor. This component accepts a list of Documents as its input - the raw text in the documents are annotated by the extractor and the annotations are stored in the document's meta dictionary (under the key named_entities).

> The component is designed to support multiple NER backends, and the current implementations support two at the moment: Hugging Face and spaCy. These two backends impelment support for any HF/spaCy model that supports token classification/NER respectively.

Reply:
Expanded the release note.